source/fma.f90

```diff
@@ -169,10 +169,11 @@ subroutine fma_postpr
   real(kind=fPrec), allocatable :: nxyzv(:,:,:)
   ! Normalised emittances
   real(kind=fPrec), allocatable :: epsnxyzv(:,:,:)
+  real(kind=fPrec) dumptas(6,6), dumpclo(6), dumptasinv(6,6)
```

Comment:
Describe the temp variable in a comment, as is done with the other variables.

Reply:
Just "Temp variables for loading normalization matrices" would suffice fine.

Reply:
Done